components/event/org.wso2.carbon.event.admin/src/main/java/org/wso2/carbon/event/admin/internal/TopicManagerAdminService.java

```diff
@@ -15,7 +15,6 @@
 /**
  * Provides topic related functions as a web service.
  */
-@Deprecated
```

Comment:
Yes since Deprecated service classes lead to java.lang.NoClassDefFoundError: javax/ws/rs/Produces error at runtime. This error is explained in following mail threads.
http://mail.wso2.org/mailarchive/dev/2012-July/008429.html
http://mail.wso2.org/mailarchive/carbon-dev/2009-November/037774.html